addons/calendar/models/calendar_recurrence.py

```diff
@@ -71,6 +71,7 @@
     ('1', 'First'),
     ('2', 'Second'),
     ('3', 'Third'),
+    ('4', 'Fourth'),
```

Comment:
I think that it was specifically removed because google calendar don't maintain this use case. 

@LucasLefevre Do you think it could be problematic (introduce unwanted behaviors/errors) in google_calendar ? 

Thanks.

Reply:
Not sure. It was done to mimic Google's UI (which seems to have changed btw). I wouldn't be surprised if this works via the API.
This is worth testing I think

Reply:
You're right @tivisse

Reply:
I try and it's working to sync with google calendar ![Odoo-s-a-Calendar-October-2020](https://user-images.githubusercontent.com/42938440/89433739-f6674d80-d742-11ea-95fa-b9b299096552.png)